astrolabe/spec_runner.py

```diff
@@ -177,7 +177,8 @@ def run(self, persist_cluster=False):
         driver_workload = json.dumps(self.spec.driverWorkload)
         worker_subprocess = subprocess.Popen([
             self.config.workload_executor, connection_string,
-            driver_workload], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            driver_workload], preexec_fn=os.setsid,
```

Comment:
Using `os.setsid` in this manner allows to ensure that all child processes of the script called via Popen inherit the process group ID of the parent. This way, we can send SIGINT to *all* child process when maintenance is done.